File: 9-palindrome-number/9-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        
        newX = str(x)
        
        print(newX[0])
        
        start = 0
        end = len(newX) - 1
        
        while start < end:
            if newX[start] != newX[end]:
                return False
            start += 1
            end -= 1
        
        return True
        

File: 9-palindrome-number/9-palindrome-number.ts
function isPalindrome(x: number): boolean {
    
    const newX = x.toString();
    
    let start = 0;
    let end = newX.length - 1;
    
    while (start < end) {
        if (newX[start] != newX[end]) {
            return false;
        } else {
            start++;
            end--;
        }
    }
    
    return true;

};